documentation/modules/adding-source-provider.adoc

```diff
@@ -15,7 +15,11 @@ EMS enforcement is disabled for migrations with VMware vSphere source providers
 
 .Prerequisites
 
-* It is strongly recommended to create a VMware Virtual Disk Development Kit (VDDK) image in a secure registry that is accessible to all clusters. A VDDK image accelerates migration and reduces the risk of a plan failing. If you are not using VDDK and a plan fails, then please retry with VDDK installed. For more information, see xref:../master.adoc#creating-vddk-image_mtv[Creating a VDDK image].
+[NOTE]
+====
+Creating a VDDK image, although optional, is highly recommended. Using {project-short} without VDDK is not recommended and could result in significantly lower migration speeds. If you are not using VDDK and a plan fails, then retry with VDDK installed. For more information, see xref:../master.adoc#creating-vddk-image_vmware[Creating a VDDK image].
```

Comment:
@RichardHoch please use the old wording `It is strongly recommended to create a VMware Vir…`

Primary reason is that we want ppl to always use it.